docs/answers-core.verticalautocompleterequest.autocompletesessionid.md

```diff
@@ -0,0 +1,13 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [@yext/answers-core](./answers-core.md) &gt; [VerticalAutocompleteRequest](./answers-core.verticalautocompleterequest.md) &gt; [autocompleteSessionId](./answers-core.verticalautocompleterequest.autocompletesessionid.md)
+
+## VerticalAutocompleteRequest.autocompleteSessionId property
+
+Used to track related autocomplete and query search for the purpose of search term aggregation.

```

Comment:
Could we say something about analytics here? I would probably say `search requests` instead of `query search`

Reply:
sure thing, updated